helpers/legacy_marshal/index.ts

```diff
@@ -91,7 +91,7 @@ function processValue(val: any, depth: number = 0): any {
   return serializedValue;
 }
 
-export function legacyMarshalValue(val: any): string | undefined {
+export function marshalValueForAnyNodeVersion(val: any): string {
```

Comment:
Do you want to rename the unmarshaler too?

Reply:
Renamed to internalUnmarshalValueForAnyNodeVersion